SYSTEMS.md

```diff
@@ -65,18 +154,28 @@ __Each item corresponds to a single column on the sheet. If you have multiple ro
   * **show** - Show the column as a **header** column.
   * **skip** - Do NOT Show the column as a **header** column.
 
-**maxwidth** - This _optional_ property controls the maximum width of the column. Value must be a number that represents the width in pixels. `"maxwidth": 200,`
+**maxwidth** - This _optional_ property controls the maximum width of the column. Value must be a number that represents the width in pixels. `"maxwidth": 200,` (note the lack of quotes around the number)
 
 **minwidth** - This _optional_ property controls the minimum width of the column. Value must be a number that represents the width in pixels. `"minwidth": 100,`
 
+**valign** - This _optional_ property controls the vertical alignment of the cells. It only accepts 'top' and 'bottom' any other css property will make it go middle. If left out, the default is inherit, so it will use whatever css the game system provides as some systems alreay override the table css and set the whole table to top, or middle. However, if your system has css code to set valign on table properties, using valign will override the system's value. 
```

Comment:
```suggestion
**valign** - This _optional_ property controls the vertical alignment of the cells. It only accepts 'top' and 'bottom'. If left out, or if an improper value is used, the setting will be ignored and will use whatever css the game system provides, since as some systems already override the table css and set the whole table to top, or middle. However, if your system has css code to set valign on table properties, using valign should override the system's value. 
```